MyNotes/Form1.cs

```diff
@@ -46,19 +38,33 @@ private void mainForm_Load(object sender, EventArgs e)
         /// <param name="e"></param>
         private void buttonCreate_Click(object sender, EventArgs e)
         {
+            const int MAX_ATTEMPTS = 10;
+
             textBoxNote.Text = "";
-            if (File.Exists(path + tag)) //если файл Unnamed.txt существует
-            {
-                File.Create(path + random.Next(0, 10).ToString() + tag); //создаём файл с рандомным числом в названии
```

Comment:
1. what if file with name like:

Unnamed3.txt 

also exists? you need check for such cases as well.

2. what if all files from Unnamed0.txt to Unnamed9.txt  exists? if we try to keep checking, we can get into infinite loop.

so, to avoid it we have two options: 
     a. increase range from random.Next(0, 10) to random.Next(0, 20) and etc.
     b. try for several times and then exit with a message.